redbot/cogs/mod/kickban.py

```diff
@@ -173,6 +173,18 @@ class KickBanMixin(MixinMeta):
             await ctx.send(_("I cannot do that due to discord hierarchy rules"))
             return
         audit_reason = get_audit_reason(author, reason)
+        if reason == None:
+            reason = "No reason was given."
+        self == self.bot.get_cog("Mod")
+        toggle = await self.settings.guild(guild).toggle_dm()
+        if toggle == True:
```

Comment:
`True` is singleton so you should either use identity check or since it's a bool, you can just use `if toggle:`
```suggestion
        if toggle:
```